backend/inspirehep/serializers.py

```diff
@@ -127,3 +130,23 @@ def _unnest_aggregations(aggregations):
                 new_aggs[agg_key] = agg_value
 
         return new_aggs
+
+    @staticmethod
+    def create_buckets_for_filter_aggregations(aggregations):
```

Comment:
would it be possible to combine this with the `_unnest_aggregations` function so you don't have to loop twice on aggregations?